lottie/src/main/java/com/airbnb/lottie/Layer.java

```diff
@@ -244,30 +244,24 @@ static Layer newInstance(JSONObject json, LottieComposition composition) {
         preCompHeight = (int) (json.optInt("h") * composition.getScale());
       }
 
-      long inFrame = json.optLong("ip");
-      long outFrame = json.optLong("op");
+      float inFrame = json.optLong("ip");
```

Comment:
This means that `inOutKeyframes` needs to change to `List<Keyframe<Integer>>`. Is that correct?